test/python/pulse/test_commands.py

```diff
@@ -22,7 +22,7 @@
 from qiskit.pulse import (SamplePulse, Acquire, FrameChange, PersistentValue,
                           Snapshot, Kernel, Discriminator, functional_pulse,
                           Delay, PulseError, ConstantPulse, Gaussian, GaussianSquare,
-                          Drag, pulse_lib)
+                          Drag, pulse_lib, SetFrequency)
```

Comment:
Since SetFrequency isn't a command, let's keep it out of this test file

Reply:
Fixed in 2cc4be3 .